softwerkskammer/lib/groupsAndMembers/groupsAndMembersService.js

```diff
@@ -134,37 +152,33 @@ module.exports = {
   },
 
   saveGroup: function saveGroup(group, callback) {
-    const self = this;
     groupsService.createOrSaveGroup(group, (err, existingGroup) => {
       if (err) { return callback(err); }
       async.each(Group.organizersOnlyInOneOf(group, existingGroup), (memberID, cb) => {
-        self.updateAdminlistSubscriptions(memberID, cb);
+        this.updateAdminlistSubscriptions(memberID, cb);
```

Comment:
Kannst Du hier nicht auch direkt this.updateAdminlistSubscriptions übergeben anstatt des selbstgemachten callbacks?